libs/common/src/helpers/date.test.ts

```diff
@@ -0,0 +1,37 @@
+import { formatInTimeZone } from 'date-fns-tz';
+import { getNowAsUtcISOString, getMonthName } from './date';
+
+describe('date helpers', () => {
+  describe('getNowAsUtcISOString', () => {
+    it('should return the current time as a UtcISOString', () => {
+      const result = getNowAsUtcISOString();
+
+      const expected = `${formatInTimeZone(new Date(), '+00:00', 'yyyy-MM-dd')}T${formatInTimeZone(new Date(), '+00:00', 'HH:mm:ss.SSS xxxxxx')}`;
+
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe('getMonthName', () => {
+    it.each([
+      { monthNumber: 1, expectedMonthName: 'January' },
+      { monthNumber: 2, expectedMonthName: 'February' },
+      { monthNumber: 3, expectedMonthName: 'March' },
+      { monthNumber: 4, expectedMonthName: 'April' },
+      { monthNumber: 5, expectedMonthName: 'May' },
+      { monthNumber: 6, expectedMonthName: 'June' },
+      { monthNumber: 7, expectedMonthName: 'July' },
+      { monthNumber: 8, expectedMonthName: 'August' },
+      { monthNumber: 9, expectedMonthName: 'September' },
+      { monthNumber: 10, expectedMonthName: 'October' },
+      { monthNumber: 11, expectedMonthName: 'November' },
+      { monthNumber: 12, expectedMonthName: 'December' },
+    ])("should return month name '$expectedMonthName' from month number '$monthNumber'", ({ monthNumber, expectedMonthName }) => {
+      // Act
+      const monthName = getMonthName(monthNumber);
+
+      // Assert
+      expect(monthName).toEqual(expectedMonthName);
+    });
```

Comment:
I am wondering if it we should change the assertion to use `format(date, 'MMMM');` 

At the moment `expectedMonthName` is essentially hardcoded/mimicking what the `format` does. 

We could simplify the `each` array to have only monthNumber and then the test can become this:

```ts
const result = getMonthName(monthNumber);

const expected = format(monthNumber, 'MMMM');
expect(result).toEqual(expectedMonthName);
```

What do you think?

Reply:
I reckon whoever wrote this code wasn't aware that `format(monthNumber, 'MMMM');` behaved that way since the implementation is quite a bit over complicated in that case. We should probably actually change the implementation to call that but then keep the tests the same?

Reply:
I don't think `const expected = format(monthNumber, 'MMMM');` will actually give you the correct month, I _think_ the first parameter is unix timestamp in millisecons so if you give it say number 2 as the month number it seems to return 'January' which is not we want.

Reply:
I don't see a why to dynamically set the expected strings that doesn't just copy the function duplication, if there was a simpler way surely we should just simplify the function implementation?

Reply:
Test feels like it is testing what date-fns does for us.

We should construct dates in the test and call the date-fns function, would be simpler and more realistic.

It is currently like testing a lodash function or an express method - no need to.

Since this has simply moved from one place to another in this PR, happy to resolve.